core-java/src/main/java/com/baeldung/ExceptionHandling/Exceptions.java

```diff
@@ -0,0 +1,251 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class Exceptions {
+	
+	 private final static Logger logger = Logger.getLogger("ExceptionLogging");
+	
+	public static void main(String[] args) throws IOException, TimeoutException {
+		
+	}
+	
+	
+	public static List<Player> getPlayers() throws IOException {
+	    Path path = Paths.get("players.dat");
+	    List<String> players = Files.readAllLines(path);
+
+	    return players.stream()
+	      .map(Player::new)
+	      .collect(Collectors.toList());
+	}
+	
+	
+	
+	public List<Player> loadAllPlayers(String playersFile) throws IOException{

```

Comment:
I'd like you to rethink these as unit tests, so that folks can run them (and so the build runs them, too). That may mean having one class called Exceptions (this class) and another called ExceptionsTests.

The method names should be descriptive regarding what is being demonstrated, instead of being called getPlayerScoreN.